ui/src/main/webapp/home/index.js

```diff
@@ -45,7 +48,10 @@ const SourceTileView = (props) => {
           <ConfigField fieldName='Endpoint' value={endpointUrl} />
           <ConfigField fieldName='Username' value={sourceUserName || 'none'} />
           <ConfigField fieldName='Password' value={sourceUserName || '******'} />
-          <RaisedButton style={{marginTop: 20}} label='Delete' secondary onClick={onDeleteConfig} />
+          <div style={{ textAlign: 'center' }}>
```

Comment:
I left this div out of the confirmable component because it isn't required for the component to work properly and I didn't want to force a centered style on anyone using the component. This div here is to center the button in the config box.